### PR TITLE
[WIP]: New UI for handling HP & NAS Jobs.

### PR DESCRIPTION
**Pull request by Akado2009**
This PR introduces new UI to submit and monitor both HP and NAS Jobs.a
Fixes: https://github.com/kubeflow/katib/issues/373

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/428)

### CODE REVIEW

File: examples/nasjob-example-RL.yaml [hunk 1]
```diff
@@ -176,4 +176,4 @@ spec:
       - name: "entropy_weight"
         value: "1e-4"
       - name: "baseline_decay"
-        value: "0.9999"
+        value: "0.9999"
```
Comment (andreyvelich) [inline examples/nasjob-example-RL.yaml]: Add newline please.

**Comment by Akado2009** [comment]
Server is now using http.Handlers, plan on moving to the router to successfully redirect to /katib since the ambassador only supports this type of routing.

File: examples/random-example.yaml [hunk 2]
```diff
@@ -34,28 +34,7 @@ spec:
...
-                  {{- with .HyperParameters}}
-                  {{- range .}}
-                  - "{{.Name}}={{.Value}}"
-                  {{- end}}
-                  {{- end}}
-                restartPolicy: Never
+      templatePath: "cpuWorkerTemplate.yaml"
   suggestionSpec:
     suggestionAlgorithm: "random"
     requestNumber: 3
```
Comment (andreyvelich) [inline examples/random-example.yaml]: No need that change

File: manifests/vizier/suggestion/nasrl/deployment.yaml [hunk 3]
```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: vizier-suggestion-nasrl
-        image: katib/suggestion-nasrl
+        image: docker.io/deepermind/katib-nasrl-suggestion
         ports:
         - name: api
           containerPort: 6789
```
Comment (andreyvelich) [inline manifests/vizier/suggestion/nasrl/deployment.yaml]: No need that change (2)

File: manifests/vizier/suggestion/nasrl/service.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: vizier-suggestion-nasrl
-  namespace: kubeflow
+  namespace: default
   labels:
     app: vizier
     component: suggestion-nasrl
```
Comment (andreyvelich) [inline manifests/vizier/suggestion/nasrl/service.yaml]: No need that change (3)

**Pull request closed without merging**